tests/tests.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+@pytest.mark.django_db
```

Comment:
Why you need  @pytest.mark.django_db ?

Reply:
For future use, you can see the usage in the next PR I uploaded.

Reply:
If it's not part of this PR, I think you should remove it and add it later

Reply:
Apparently, it's necessary for the tests that I wrote there I removed that and got an error that explicitly says that it is required to use that annotation for those tests.
@natifridman @yulisuliman